pages/common/aws-codecommit.md

```diff
@@ -0,0 +1,20 @@
+# aws codecommit
+
+> AWS CodeCommit is a secure, highly scalable, managed source control service that hosts private Git repositories.
+> More information: <https://aws.amazon.com/codecommit/>.
+
+- View a list of all subcommands:
+
+`aws codecommit help`
+
+- Configure the AWS CLI with a profile:
+
+`aws configure`
+
+- Display help:
```

Comment:
Move this command to last

Reply:
@kbdharun I have moved the command to the last you can check now